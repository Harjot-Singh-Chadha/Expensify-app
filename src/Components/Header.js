import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <div>
    
     <h2>Expensify</h2>
     <NavLink to='/' activeClassName='is-active' exact={true}>Dashboard</NavLink>&nbsp;
     <NavLink to='/create' activeClassName='is-active'>Create</NavLink>&nbsp;
    
     
    </div>
    
 );

 export default Header