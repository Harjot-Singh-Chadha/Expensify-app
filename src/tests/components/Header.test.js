import React from 'react';
import { shallow } from 'enzyme';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../Components/Header';
import toJSON from 'enzyme-to-json';

test('should render header', () => {
   const wrapper = shallow(<Header />);
   expect(wrapper).toMatchSnapshot();

  // expect(wrapper.find('h1').text()).toBe('Expensify');
  // const renderer = new ReactShallowRenderer();
  // renderer.render(<Header />);
  // expect(renderer.getRenderOutput()).toMatchSnapshot();

});