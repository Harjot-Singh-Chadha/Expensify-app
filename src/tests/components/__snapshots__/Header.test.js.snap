// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header 1`] = `
<div>
  <h2>
    Expensify
  </h2>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
   
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create
  </NavLink>
   
</div>
`;
